updates.js

```diff
@@ -4482,9 +4483,13 @@ function toggleSetting(setting, elem, fromPortal, updateOnly, backwards){
 				var tierValue = "<span style='color: black;' class='" + achievement.icon + "'></span>";
 				if ((!one && achievement.finished == x) || (one && !achievement.finished[x] && game.global.highestLevelCleared >= achievement.filters[x])) {
 					if (item == "humaneRun")
-						displayColor = (achievement.evaluate() == 0) ? "#b32d00" : "#C5C515"; //Yellow
+						displayColor = (achievement.evaluate() == 0) ? "#b32d00" : "#C5C515"; // Red / Yellow
+					else if(one)
+						displayColor = checkFeatEarnable(achievement.names[x]) ? "#C5C515" : "#b32d00"; // Yellow / Red
+					else if(achievement.timed && minutes > achievement.breakpoints[x])
```

Comment:
```suggestion
					else if(achievement.timed && minutes >= achievement.breakpoints[x])
```
Minutes indeed should be `gte` as seen in `checkAchieve()`:

https://github.com/Trimps/Trimps.github.io/blob/ceb78776846870de5d56caf1354401280438570d/updates.js#L4421